labs/test/tsc/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+    "compilerOptions": {
+        "strict": true,
+        "target": "es2015",
+        "module": "commonjs",
+        "declaration": true,
+        "sourceMap": true,
+        "rootDirs": [
+            ".",
+            "../../../bazel-out/darwin-fastbuild/bin/labs/test/tsc",
```

Comment:
This is needed for tsc to work as it needs to find .ts files from the source tree and .d.ts files from the output tree. Obviously not portable like this, you need to include all of the `<platform-fastbuild>` and `<platform-debug>` etc... in the list and it would be a pain to maintain.

I'm considering generating a `{name}_tsconfig.json` in the rule that extends from the user tsconfig and just adds just adds these `rootDirs`.